### PR TITLE
✨ feat - organise rules by categories + display a dashboard by category in the results

### PR DESCRIPTION
**Pull request by PChol22**

### CODE REVIEW

**Comment by PChol22** [comment]
image

**Comment by PChol22** [comment]
Screenshot 2022-09-09 at 16 13 00

**Comment by PChol22** [comment]
Screenshot 2022-09-09 at 16 13 40

**Review by ealain — APPROVED** [review]
Good job!

File: src/types/CheckResult.ts [hunk 1]
```diff
@@ -19,3 +20,46 @@ export type ResourceResult = {
...
+  [Category.GREEN_IT]: 'Green IT',
+  [Category.STABILITY]: 'Stability',
+  [Category.SPEED]: 'Speed',
+  [Category.IT_COSTS]: 'IT costs',
+  [Category.SECURITY]: 'Security',
+};
+
+export const RulesCategories: Record<Rules, Category[]> = {
+  [Rules.ASYNC_SPECIFY_FAILURE_DESTINATION]: [Category.STABILITY],
+  [Rules.INTELLIGENT_TIERING]: [Category.GREEN_IT, Category.GREEN_IT],
```
Comment (ealain) [inline src/types/CheckResult.ts:42]: `Category.IT_COSTS`

**Pull request merged**